src/midi/midi.cpp

```diff
@@ -246,8 +246,13 @@ class MIDI : public Module_base {
 				}
 				midi.handler=handler;
 				midi.available=true;
-				LOG_MSG("MIDI: Opened device: %s",
-				        handler->GetName());
+				const std::string name = handler->GetName();
+				if (name == "none")
+					LOG_MSG("MIDI: No device selected - output disabled");
```

Comment:
In the case when the user explicitly sets `mididevice = none`, it would be nice to not print anything about MIDI, which is consistent when other audio devices are similarly disabled. 

But I agree that we should print something like this if the user has set `mididevice = something`, but we can't find a workable device or connection, and hit the end of the handler linked list.

Reply:
You are right - I shouldn't complicate this function even more; I will drop this commit from this PR and take a stab at addressing this in the future PR.

Reply:
(I dropped the change for now - to be addressed in another PR)